packages/graphql-hooks/src/useClientRequest.ts

```diff
@@ -1,4 +1,6 @@
 import React from 'react'
+import { TypedDocumentNode } from '@graphql-typed-document-node/core'
```

Comment:
I have a weird dubt now, after seeing that you added this as a dev dependency, although it's used in production code. unless there's one of the various bundles that we generate which is not actually a bundle but requires the deps to be installed, is there a chance that in fact this package doesn't need any production dependencies whatsoever? if unclear, let's talk :)

Reply:
🤔 Yes I think you're right, this should be a production dependency. In the sense that 'production' is someone installing this library to develop with then it'll be required.

Reply:
if somebody is installing this library for developing, will they really need it? on the other hand, if somebody is installing (which is the usual case) this library for _using_ it, I suspect they'll get all of the production deps, which aren't quite necessary I suppose

Reply:
The type will be required when developing or using the library in order to get the result type information. It is superfluous if the consumer isn't using `DocumentNode`s though

Reply:
ah so it does need to be a production dependency, unless the type information ends up in the bundle, but I suspect it doesn't